meetings/2024-09-27.md

```diff
@@ -134,8 +139,8 @@ URL: https://lists.w3.org/Archives/Public/public-lws-wg/2024Sep/0002.html
+* SC: CG have some issues on this already. There's an ocean of issues. And, as HZ pointed out portability, we have issues related to that as well. And on provisioning a data storage ( https://github.com/solid/specification/issues/317 ). Some of that had some question marks. For example, there's a proposal made by PDS Interop folks for [Solid Migrator](https://github.com/pdsinterop/solid-migrator) submitted to CG - by MdJ in https://github.com/solid/specification/issues/474 , https://github.com/pdsinterop/solid-link-metadata - including implementation experience, but we did not do a good job evaluating that, having to do with redirects, deleting, moving. See also https://github.com/solid/specification/issues/136 , https://github.com/solid/specification/issues/462 . Some of that can be evaluated in the CG still. There's work that already had been done and if it seems mature we can handover to WG anytime. In the CG we have plenty of threads/considerations put into our radar and we need to revisit instead of assuming it's not there. 
+* SC: I find this portability topic comes up a lot but an important distinction is what web architecture says about moving resources around, it's very clear about redirects. Another topic about moving data around. AFAIK those are separate things: moving URIs around than having a copy of your data in a new storage. The URI aspect is covered under webarch, copying/archiving your storage could be exporting a dump of your storage and having mechanism on the new storage to take that and extract, for example. There is even `HTTP COPY` (from WebDAV) and we had that in our radar ( https://github.com/solid/specification/issues/19 ). Copying the data is different from saying this resource identified by this URI is moved to another location. The harder parts of the problem is coming up with URI persistence policies and enforcing. What kind of assurances will there be that they can continue to use that when moved to another storage? Some of that is the policy work and not so much the architectural decision on how protocols and interfaces speak to each other. If we can find persistence policies that'd be the answer to the problem. The copying the data seems more complex technically but it's a lot simpler than handling the persistence issues. 
```

Comment:
```suggestion
* SC: The CG has some issues on this already. There's an ocean of issues. And, as HZ pointed out — portability; we have issues related to that as well. And on provisioning a data storage <https://github.com/solid/specification/issues/317>. Some of that had some question marks. For example, there's a proposal made by PDS Interop folks for [Solid Migrator](https://github.com/pdsinterop/solid-migrator), submitted to CG by MdJ in <https://github.com/solid/specification/issues/474>, <https://github.com/pdsinterop/solid-link-metadata> — including implementation experience, but we did not do a good job evaluating that, having to do with redirects, deleting, moving. See also <https://github.com/solid/specification/issues/136>, <https://github.com/solid/specification/issues/462>. Some of that can still be evaluated in the CG. There's work that has already been done, and if it seems mature, we can hand it over to the WG, anytime. In the CG, we have plenty of threads/considerations put on our radar, and we need to revisit them instead of assuming it's not there. 
* SC: I find this portability topic comes up a lot, but an important distinction is what [Web Architecture](https://www.w3.org/TR/webarch/) says about moving _resources_ around — it's very clear about redirects. Moving _data_ around is another topic. AFAIK those are separate things: moving URIs around vs. having a copy of your data in a new storage. The URI aspect is covered under webarch; copying/archiving your storage could be exporting a dump of your storage and having mechanisms on the new storage to take that and extract, for example. There is even `HTTP COPY` (from WebDAV), and we had that on our radar <https://github.com/solid/specification/issues/19>. Copying the data is different from saying "this resource identified by this URI is moved to another location". The harder parts of the problem are coming up with URI persistence policies and then enforcing them. What kind of assurances will there be that they can continue to use that when moved to another storage? Some of that is policy work and not so much the architectural decision of how protocols and interfaces speak to each other. If we can find persistence policies, that'd be the answer to the problem. Copying the data seems more complex technically, but it's a lot simpler than handling the persistence issues. 
```